tools/picard.js

```diff
@@ -377,7 +385,8 @@ function connect_signals () {
     SetMap.init_map_with_models(sets);
 
     widgets.add_box.connect('clicked', () => {
-        let model = Eknc.ArticleObjectModel.new_from_props({
+        let object_model = widgets.object_model_combo_box.get_active_text();
```

Comment:
I guess not, but I want to be able to have different object models at the same time